### ISSUE TITLE
Unfixable u2u1011

### ISSUE THREAD

**Issue by wmjordan** [u2uconsult/codeanalyzers#35]
Given the following code snippet, u2u1011 is triggered "`Method Test.GetData(long) can return type IEnumerable<<anonymous type: int i, DateTime d>>`", but the code can not be fixed since it is not possible to return an anonymous type.

````C#
public object GetData() {
    return new int[] { 1, 3, 5 }.Select(i => new { i, d = DateTime.Now });
}
````

**Comment by KrisVandermotten** [comment]
This bug is fixed in version 3.8.5. Thanks for reporting!
